nixos/tests/nextcloud/with-declarative-redis-and-secrets.nix

```diff
@@ -1,6 +1,6 @@
 args@{ nextcloudVersion ? 27, ... }:
 (import ../make-test-python.nix ({ pkgs, ...}: let
-  username = "custom_admin_username";
+  adminuser = "root";
```

Comment:
Was this perhaps part of your debugging that you accidentally pushed? Apparently the issue was that we need a different URL now for rclone.
Also, I think that this is somewhat useful, i.e. to have the case tested with a non-standard admin user.